libs/sdk-backend-tiger/src/backend/workspace/insights/mocks/visualizationClasses.ts

```diff
@@ -100,6 +100,30 @@ export const visualizationClasses: IVisualizationClass[] = sortBy(
                 uri: "/gdc/md/gtl83h4doozbp26q0kf5qg8uiyu4glyn/obj/8",
             },
         },
+        {
```

Comment:
BTW what's the use of these mocks?

Reply:
On tiger, there's nothing like preloaded objects -- visualization classes are hardcoded here and not retrieved via api from the platform.